uamqp/cbs.py

```diff
@@ -18,29 +18,27 @@
 from .constants import (
     CbsState,
     CbsAuthState,
-    AUTH_TIMEOUT,
-    TOKEN_TYPE_SASTOKEN,
     CBS_PUT_TOKEN,
     CBS_EXPIRATION,
     CBS_NAME,
     CBS_TYPE,
     CBS_OPERATION,
     ManagementExecuteOperationResult,
-    ManagementOpenResult
+    ManagementOpenResult,
+    AUTH_TIMEOUT,
+    TOKEN_TYPE_SASTOKEN
 )
 
 _LOGGER = logging.getLogger(__name__)
 
 
-class CbsAuth(object):
+class CBSAuthenticator(object):
     def __init__(
-            self,
-            session,
-            auth_audience,
-            get_token,
-            token_type=TOKEN_TYPE_SASTOKEN,
-            auth_timeout=AUTH_TIMEOUT,
-            encoding='UTF-8'
+        self,
+        session,
+        auth,
+        auth_timeout=AUTH_TIMEOUT,
+        encoding='UTF-8'
```

Comment:
I think we can strip out the encoding support. I believe CBS should always be utf-8, we can simply the code by simply removing this. Or not decode at all.